.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,53 @@
+name: Spellcheck
+
+on: [pull_request]
+
+jobs:
+  spellcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with: 
+          node-version: '14.x'
+
+      - name: Spellcheck
+        uses: lingeorge88/amplify-ghActions-tests
+        with:
+          path: '.'
+
+      - name: Post spellcheck results
+        uses: actions/github-script@v4
+        with:
+          github-token: ${{ secrets.GH_TOKEN }}
+          script: |
+            const fs = require('fs')
+            const context = github.context
+            const octokit = github.getOctokit(process.env.GITHUB_TOKEN)
```

Comment:
I think the idea here to check every file for spelling mistakes is pretty interesting, but I am not quite sure I understand _how_ it is checking each file. What is the source against which we check out code? How can we ensure that it will not accidentally change JavaScript language keywords?